components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -650,7 +650,12 @@ abstract class AbstractFetchDownloadService : Service() {
         }
 
         var isUsingHttpClient = false
-        val request = Request(download.url.sanitizeURL(), headers = headers, cookiePolicy = cookiePolicy)
+        val request = Request(
+            download.url.sanitizeURL(),
+            headers = headers,
+            cookiePolicy = cookiePolicy,
+            private = download.private
```

Comment:
My assumption is that we can now remove the cookie policy here now and allow the inclusion of cookies, since in the private mode they will not survive the session?

Reply:
Yes, that's that correct.

Reply:
Done!

Reply:
If there anything else just let me know :)